spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -1239,12 +1238,9 @@ function timeoutPacket(
       // NOTE: For ORDERED_ALLOW_TIMEOUT, the relayer must first attempt the receive on the destination chain
       // before the timeout receipt can be written and subsequently proven on the sender chain in timeoutPacket
       case ORDERED_ALLOW_TIMEOUT:
-        // ordered channel: check that packet has not been received
-        // only allow timeout on next sequence so all sequences before the timed out packet are processed (received/timed out)
-        // before this packet times out
-        abortTransactionUnless(nextSequenceRecv == packet.sequence)
+        abortTransactionUnless(packet.sequence < nextSequenceRecv)
```

Comment:
@AdityaSripal if a channel is open and the relayer must first attempt to receive the packet on destination chain, I guess it is ok to assume that `nextSequenceRecv` will have been incremented and therefore is > `packet.sequence`?

Reply:
No we will actually want the equals here because I think it makes sense to enforce that timeout happens in order as well.

Reply:
Should the condition be then `abortTransactionUnless(packet.sequence == nextSequenceRecv - 1)` since `nextSequenceRecv` will have been incremented when attempting to receive packet with `packet.sequence`?